projects/addon-commerce/components/money/money.template.html

```diff
@@ -19,7 +19,7 @@
             [textContent]="value | tuiFractionPart: decimal : precision"
         ></span>
     </span>
-    <span *ngIf="alignRight || currencySymbol.length > 1">&nbsp;</span>
+    <span *ngIf="currencySymbol.length">&nbsp;</span>
```

Comment:
no, there should not be a space if currency alignLeft and currency is just one symbol

`+$123` but `+MXN{space}123`

so the condition must be alignRight && currencySymbol || currencySymbol.length > 1

Reply:
let's also add to unit 

```
it(`if empty string is passed, there is no extra space character`, () => {
            testComponent.currency = ``;
            fixture.detectChanges();

            const textContent = fixture.debugElement.nativeElement.textContent;

            expect(textContent).toEqual(`237`);
        });
```

Reply:
What's the point of checking length on the right side?

Reply:
(by design)

Reply:
fixed